### PR TITLE
Add Onboarding Resource Roundup Blog post

### PR DESCRIPTION
**Pull request by saragon02**
Request review from @fenekku, @lnielsen, @jbenito3

### CODE REVIEW

File: content/blog/2021-02-10-onboarding-resource-roundup/contents.lr [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+body:
+
+Congratulations! Your team has just joined the InvenioRDM open source project and is eager to contribute as much as possible. With a mixture of dedicated developers, resource curators, subject matter experts, and various leadership stakeholders, your team simply needs to know about the available resources and how to get started. Read on!
+
+
+### Developers
+
+Whether you would like to install and customize InvenioRDM, contribute on the main project sprints, or take on development of a module locally to contribute back to the project, there are a few important points of entry to participation:
+
+- [Roadmap](https://inveniosoftware.org/products/rdm/roadmap/): Familiarize yourself with the overall project timeline and goals. Stable releases will be made through Summer 2021 on a monthly/bi-monthly basis
```
Comment (fenekku) [inline content/blog/2021-02-10-onboarding-resource-roundup/contents.lr:14]: Just "monthly-basis" is good enough I think. It's a more stable commitment (less wishy-washy) and we are upholding it.

File: content/blog/2021-02-10-onboarding-resource-roundup/contents.lr [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+
+- [Development Roadmap](https://github.com/inveniosoftware/product-rdm/milestones?direction=asc&sort=due_date&state=open): Get to know the specific milestones for 2021
+
+- [Sprintboard](https://github.com/orgs/inveniosoftware/projects/63): See the planning board for the current sprint
+
+- [Documentation](https://inveniordm.docs.cern.ch/): Read the official project documentation
+
+- [Discord](https://discord.gg/8qatqBC): Ready to get started? Ask Lars for an invitation to the project Discord (chat/video platform) and introduce yourself in either the general or newcomers-help channel. A daily developer standup takes place on Discord at 15:15 UTC
```
Comment (fenekku) [inline content/blog/2021-02-10-onboarding-resource-roundup/contents.lr:22]: I am just wondering here if people must go through Discord to "get started"... Something like:  

```suggestion
- [Discord](https://discord.gg/8qatqBC): Ask questions in the project Discord (chat/video platform) and introduce yourself in either the general or newcomers-help channels. A daily developer standup takes place on Discord at 15:15 UTC
```
? This way they don't flood Lars and feel like they can get to work without ceremony ?

File: content/blog/2021-02-10-onboarding-resource-roundup/contents.lr [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+
+- [Development Workflows](https://inveniordm.docs.cern.ch/maintenance/development_workflow/): Learn about the development workflows used
+
+### Community Members
+
+All stakeholders at your institution are welcome to contribute to InvenioRDM, whether their contribution is code-based or not. To create a tool to best serve the community, outlining stakeholder requirements will be vital:
+
+ - [Demo site](https://inveniordm.web.cern.ch/): Access the latest sandbox version of InvenioRDM here. Create an account to test record creation
+
+ - [Discourse](https://invenio-talk.web.cern.ch/): Provide feedback on testing here. Use the [Feedback](https://invenio-talk.web.cern.ch/c/site-feedback/2) category or start a new thread wherever you like
```
Comment (fenekku) [inline content/blog/2021-02-10-onboarding-resource-roundup/contents.lr:36]: Hmm... I am only going to check the monthly feedback thread referred above for time (quicker), specificity (know which release) and singularity (if same issue encountered multiple times, others can see it in the same thread). I also think the feedback section is for the framework as a whole (i.e. not just InvenioRDM).

Could be fine if others are checking these threads, but it's setting up additional work ;)

File: content/blog/2021-02-10-onboarding-resource-roundup/contents.lr [hunk 4]
```diff
@@ -0,0 +1,67 @@
...
+
+ - Welcome, Help and General Questions: Reach out to Community Manager Sara Gonzales via [Discord](https://discordapp.com/users/saragon7740) for general questions or to join the Metadata Interest Group
+
+ - Schedule a Demo: Is your site, or a site you know, considering implementing InvenioRDM? Reach out to Community Manager Sara Gonzales via [Discord](https://discordapp.com/users/saragon7740) to schedule a one-hour demo and information session about the repository
+ 
+
+ ### Everyone
+
+ - Project list-serv: Lars will have shared this email address with you upon joining the team. This is the frontline communication tool for the InvenioRDM community. Make this email address a safe sender to receive the latest updates on Telecons and community-based information such as invitations to the Metadata Interest Group meetings. Polls, forms, and additional community tools will be sent to this list-serv
```
Comment (fenekku) [inline content/blog/2021-02-10-onboarding-resource-roundup/contents.lr:49]:
```suggestion
 - Project list-serv: Lars will have shared this email address with you upon joining the team. This is the frontline communication tool for the InvenioRDM community. Make this email address a safe sender to receive the latest updates on teleconferences (a.k.a. telecons) and community-based information such as invitations to the Metadata Interest Group meetings. Polls, forms, and additional community tools will be sent to this list-serv
```

File: content/blog/2021-02-10-onboarding-resource-roundup/contents.lr [hunk 5]
```diff
@@ -0,0 +1,67 @@
...
+
+ - Telecons (see the list-serv for connection information): All partners of the InvenioRDM community are invited to bi-weekly Telecons throughout the year. One Telecon per month will focus on development updates, and the other will focus on community-based topics
```
Comment (fenekku) [inline content/blog/2021-02-10-onboarding-resource-roundup/contents.lr:51]: Telecon -> telecon (is there a reason for the casing?)

File: content/blog/2021-02-10-onboarding-resource-roundup/contents.lr [hunk 6]
```diff
@@ -0,0 +1,67 @@
...
+
+ - [InvenioRDM Telecon Schedule 2021](https://docs.google.com/document/d/1Ks5_xEMujd6vdlSmYGrbCsKP_I2hZasMkg-RIjwxbKg/edit?usp=sharing)
+
+ - [Blog](https://inveniosoftware.org/blog/): The blog will also alternate between development-based and community-based topics. Would you like to contribute a blog post? [Sign up here](https://docs.google.com/document/d/17Zupy06bKPdJMiLJDsfsgKZLgRJnbuj70AUCY1bOA0w/edit?usp=sharing) or contact Sara to get on the schedule
```
Comment (fenekku) [inline content/blog/2021-02-10-onboarding-resource-roundup/contents.lr:55]:
```suggestion
 - [Blog](https://inveniosoftware.org/blog/): You are reading it! The blog will also alternate between development-based and community-based topics. Would you like to contribute a blog post? [Sign up here](https://docs.google.com/document/d/17Zupy06bKPdJMiLJDsfsgKZLgRJnbuj70AUCY1bOA0w/edit?usp=sharing) or contact Sara to get on the schedule
```

**Review by fenekku — APPROVED** [review]
A couple of comments, but it's good to merge after you take a look (you judge if those are needed).

File: content/blog/2021-02-10-onboarding-resource-roundup/contents.lr (hunk 1 above)
Comment (saragon02) [reply content/blog/2021-02-10-onboarding-resource-roundup/contents.lr:14]: Agreed - I'll make this update.

File: content/blog/2021-02-10-onboarding-resource-roundup/contents.lr (hunk 3 above)
Comment (saragon02) [reply content/blog/2021-02-10-onboarding-resource-roundup/contents.lr:36]: Good to know. Are others checking? I could also check once/month and alert you to any posts.

File: content/blog/2021-02-10-onboarding-resource-roundup/contents.lr (hunk 5 above)
Comment (saragon02) [reply content/blog/2021-02-10-onboarding-resource-roundup/contents.lr:51]: Updated to lower case

File: content/blog/2021-02-10-onboarding-resource-roundup/contents.lr (hunk 2 above)
Comment (saragon02) [reply content/blog/2021-02-10-onboarding-resource-roundup/contents.lr:22]: Sounds good!

File: content/blog/2021-02-10-onboarding-resource-roundup/contents.lr (hunk 4 above)
Comment (saragon02) [reply content/blog/2021-02-10-onboarding-resource-roundup/contents.lr:49]: Sounds good!

File: content/blog/2021-02-10-onboarding-resource-roundup/contents.lr (hunk 3 above)
Comment (fenekku) [reply content/blog/2021-02-10-onboarding-resource-roundup/contents.lr:36]: I don't know if others are checking it regularly. Discourse sends a weekly digest email if you haven't visited the site in the last week to keep you appraised of posts I believe. The responsiveness is therefore not as quick, but the feedback is more permanent and focused than on Discord (I know Discord and Discourse have similar names...).GitHub tickets are more technical. In the end we have feedback areas for everyone, but it's hard for developers to keep track of them. That's why I try to limit myself...

**Comment by fenekku** [comment]
@lnielsen this should be accompanied by a push to `maint` (or equivalent branch for this project) to release the blog post. It would also leave you the ability to do last minute modifications.

**Pull request merged**